entry.go

```diff
@@ -52,6 +52,10 @@ func (e *Entry) WithField(key string, value interface{}) *Entry {
 // The given error may implement .Fielder, if it does the method
 // will add all its `.Fields()` into the returned entry.
 func (e *Entry) WithError(err error) *Entry {
+	if e == nil {
```

Comment:
Shouldn't this be `if err == nil {` or am I misunderstanding the intent?

Reply:
Agreed, it should be.